### PR TITLE
[Hotfix]: Sync Test Id In All Tests

### PR DESCRIPTION
**Pull request by AmirabbasJ**
This PR syncs test id in all tests.

### CODE REVIEW

**Review by ASafaeirad — APPROVED** [review]

**Comment by erfan-goodarzi** [comment]
@AmirabbasJ Oops man.... I was doing exactly the same thing.

File: cypress/e2e/createProject.cy.ts [hunk 1]
```diff
@@ -13,21 +13,23 @@ describe('Create Project', () => {
...
 
   it('Finds the create project button in projects page', () => {
-    cy.get(createProjectButtonSelector).should('exist');
+    cy.findByTestId(createProjectId).should('exist');
   });
 
   it('Finds the modal after clicking on the create project button', () => {
-    cy.get(createProjectButtonSelector).click();
-    cy.get(createProjectModalSelector).should('exist');
+    cy.findByTestId(createProjectId).click();
```
Comment (erfan-goodarzi) [inline cypress/e2e/createProject.cy.ts:20]: The same thing here. the purpose of this test is to have a button and click on it to show something. so u just repeat them in every `it` function.

I think before each can be like this:
```
beforeEach(() => {
    cy.visit('/');
    cy.get(createProjectNavSelector).click();
    cy.findByTestId(createProjectId).should('exist');
    cy.findByTestId(createProjectId).click();
  });
```

@ASafaeirad What do u think?

File: src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx [hunk 2]
```diff
@@ -1,13 +1,11 @@
 import { messages } from '@camp/messages';
 import { noop } from '@fullstacksjs/toolbox';
 
+import { createProjectFormIds } from './CreateProjectForm';
 import { CreateProjectModal } from './CreateProjectModal';
 
 const requiredFieldMessage = messages.projects.validation.required;
```
Comment (erfan-goodarzi) [inline src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx:7]: This way is better.
```suggestion
const { minLength: minLengthMessage, required: requiredFieldMessage } =
  messages.families.validation;
```

**Review by erfan-goodarzi — CHANGES_REQUESTED** [review]

**Comment by AmirabbasJ** [comment]
> @AmirabbasJ Oops man.... I was doing exactly the same thing.

my bad, I hold this one without doing a pr for like 3 days or something

File: cypress/e2e/createProject.cy.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
+import {
+  createProjectFormIds,
+  createProjectId,
```
[the remark this answers is not in the document]
Comment (AmirabbasJ) [reply cypress/e2e/createProject.cy.ts:3]: yes that more explicit

File: cypress/e2e/createProject.cy.ts [hunk 4]
```diff
@@ -13,21 +13,23 @@ describe('Create Project', () => {
   });
 
   it('Finds the create project button in projects page', () => {
-    cy.get(createProjectButtonSelector).should('exist');
+    cy.findByTestId(createProjectId).should('exist');
```
[the remark this answers is not in the document]
Comment (AmirabbasJ) [reply cypress/e2e/createProject.cy.ts:16]: that would be correct if we did not had `Finds the create project button in projects page` test in our tests
it is by itself a test case so we can't do that

File: cypress/e2e/createProject.cy.ts (hunk 1 above)
Comment (AmirabbasJ) [reply cypress/e2e/createProject.cy.ts:20]: like I said in the previous comment 
each of these lines are a test of their own
as far as I can tell, what we put in the `beforeEach` is kinda our assumptions, we say this beforeEach holds for every test
so what's the point of saying that X holds before all these tests run but also have test for X in the test too
this is kinda of a paradox

what u could suggest that can work is to not have test for these stuff at all and just assume they hold for every test and then put them in `beforeEach` which is up to discussion

File: src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx (hunk 2 above)
Comment (ASafaeirad) [reply src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx:7]: Me neither

Comment (AmirabbasJ) [reply src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx:7]: ok guys it seems I did it somehow xD
it used to be this
```ts
const requiredFieldMessage = 'این فیلد ضروری است';
const minLengthMessage = 'نام خانواده باید حداقل ۳ حرف باشد';
```
and I just ran a find and replace in vscode and it turned out this way:
```ts
const requiredFieldMessage = messages.families.validation.required;
const minLengthMessage = messages.families.validation.minLength;
```

Comment (erfan-goodarzi) [reply src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx:7]: What's the problem with this?

**Comment by erfan-goodarzi** [comment]
> my bad, I hold this one without doing a pr for like 3 days or something

Don't mention it bro. u did the right thing

**Review by erfan-goodarzi — APPROVED** [review]

**Pull request merged**